assets/css/main.css

```diff
@@ -136,6 +140,13 @@ textarea {
   margin-left: 19px;
 }
 
+.margin-right-19 {
```

Comment:
margin-right-19 don't follow any design system or principle. If we want to use arbitrary values like this, we can follow the normal css conventions by creating classes and adding properties.

Reply:
resolved in different branch